src/main/java/bio/terra/cli/apps/LocalProcessCommandRunner.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.cli.apps;
+
+import bio.terra.cli.apps.utils.LocalProcessLauncher;
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** This class runs client-side tools in a local process. */
+public class LocalProcessCommandRunner extends CommandRunner {
+  private static final Logger logger = LoggerFactory.getLogger(LocalProcessCommandRunner.class);
+
+  private LocalProcessLauncher localProcessLauncher = new LocalProcessLauncher();
+
+  public LocalProcessCommandRunner(GlobalContext globalContext, WorkspaceContext workspaceContext) {
+    super(globalContext, workspaceContext);
+  }
+
+  /**
+   * Run a tool command in a local process.
+   *
+   * <p>Some setup/cleanup commands for gcloud authentication and configuration will be run
+   * before/after the given command.
+   *
+   * @param command the command and arguments to execute
+   * @param envVars a mapping of environment variable names to values
+   */
+  protected void runToolCommandImpl(List<String> command, Map<String, String> envVars) {
+    List<String> bashCommands = new ArrayList<>();
+    bashCommands.add("gcloud config set project " + workspaceContext.getGoogleProject());
+    bashCommands.add(
```

Comment:
Yes, analogous. I hesitate to re-use the same script because I think it's likely we'll want to do different things for docker vs local process. But adding this method to the abstract class sounds like a good idea. I'll give it a shot.